wezterm-gui/src/overlay/copy.rs

```diff
@@ -591,6 +591,35 @@ impl CopyRenderable {
         self.select_to_cursor_pos();
     }
 
+    fn move_to_selection_other_end(&mut self) {
+        if let Some(old_start) = self.start {
+            // Swap cursor & start of selection
+            self.start.replace(SelectionCoordinate {
+                x: self.cursor.x,
+                y: self.cursor.y,
+            });
+            self.cursor.x = old_start.x;
+            self.cursor.y = old_start.y;
+            self.select_to_cursor_pos();
+        }
+    }
+
+    fn move_to_selection_other_end_horiz(&mut self) {
+        if self.selection_mode != SelectionMode::Block {
+            // This action makes sense ONLY in block mode
+            return;
+        }
```

Comment:
vim appears to treat this case as `move_to_selection_other_end`; should we do the same?

```suggestion
        if self.selection_mode != SelectionMode::Block {
            return self.move_to_selection_other_end();
        }
```